terraform/monitoring/config/alert.rules.yml

```diff
@@ -3,7 +3,7 @@ groups:
   rules:
   - alert: ProdRequestsFailuresElevated
     # Condition for alerting
-    expr: sum(rate(requests{app="teaching-vacancies-production", status_range=~"0xx|4xx|5xx"}[5m]))*60 > 25
+    expr: sum(rate(requests{app="teaching-vacancies-production", status_range=~"0xx|4xx|5xx"}[5m]))*60 > 40
```

Comment:
25 looks reasonable?

Reply:
Yes, I agree. Have left it at 25. 
Looks like elsewhere we've identified the source of 4 additional HTTP failures with missing files